packages/alfa-selector/src/selector.ts

```diff
@@ -1234,7 +1234,10 @@ export namespace Selector {
       element: Element,
       context: Context = Context.empty()
     ): boolean {
-      return context.isHovered(element);
+      return element
+        .inclusiveDescendants(Node.fullTree)
+        .filter(isElement)
+        .some((element) => context.isHovered(element));
```

Comment:
```suggestion
        .some(context.isHovered);
```
This can likely be eta-contracted.